Lib/ufo2ft/__init__.py

```diff
@@ -1,582 +1,34 @@
-      is missing or True (default), the glyphs are renamed. Set to False
-      to keep the original names.
-
-    **inplace** (bool) specifies whether the filters should modify the input
-      UFO's glyphs, a copy should be made first.
-
-    *layerName* specifies which layer should be compiled. When compiling something
-    other than the default layer, feature compilation is skipped.
-
-    *skipExportGlyphs* is a list or set of glyph names to not be exported to the
-    final font. If these glyphs are used as components in any other glyph, those
-    components get decomposed. If the parameter is not passed in, the UFO's
-    "public.skipExportGlyphs" lib key will be consulted. If it doesn't exist,
-    all glyphs are exported. UFO groups and kerning will be pruned of skipped
-    glyphs.
-
-    *cffVersion* (int) is the CFF format, choose between 1 (default) and 2.
-
-    *subroutinizer* (Optional[str]) is the name of the library to use for
-      compressing CFF charstrings, if subroutinization is enabled by optimizeCFF
-      parameter. Choose between "cffsubr" or "compreffor".
-      By default "cffsubr" is used for both CFF 1 and CFF 2.
-      NOTE: cffsubr is required for subroutinizing CFF2 tables, as compreffor
-      currently doesn't support it.
-    """
-    kwargs = init_kwargs(kwargs, compileOTF_args)
-    glyphSet = call_preprocessor(ufo, **kwargs)
-
-    logger.info("Building OpenType tables")
-    optimizeCFF = CFFOptimization(kwargs.pop("optimizeCFF"))
-    tables = kwargs.pop("_tables")
-    otf = call_outline_compiler(
-        ufo,
-        glyphSet,
-        **kwargs,
-        optimizeCFF=optimizeCFF >= CFFOptimization.SPECIALIZE,
-        tables=tables,
-    )
-
-    # Only the default layer is likely to have all glyphs used in feature code.
-    if kwargs["layerName"] is None:
-        compileFeatures(ufo, otf, glyphSet=glyphSet, **kwargs)
-
-    return call_postprocessor(
-        otf,
-        ufo,
-        glyphSet,
-        **kwargs,
-        optimizeCFF=optimizeCFF >= CFFOptimization.SUBROUTINIZE,
-    )
-
-
-compileTTF_args = {
-    **base_args,
-    **dict(
-        preProcessorClass=TTFPreProcessor,
-        outlineCompilerClass=OutlineTTFCompiler,
-        convertCubics=True,
-        cubicConversionError=None,
-        reverseDirection=True,
-        rememberCurveType=True,
-        flattenComponents=False,
-        autoUseMyMetrics=True,
-        dropImpliedOnCurves=False,
-        allQuadratic=True,
-    ),
-}
+from ufo2ft._compilers.interpolatableOTFCompiler import InterpolatableOTFCompiler
+from ufo2ft._compilers.interpolatableTTFCompiler import InterpolatableTTFCompiler
+from ufo2ft._compilers.otfCompiler import OTFCompiler
+from ufo2ft._compilers.ttfCompiler import TTFCompiler
+from ufo2ft._compilers.variableCFF2sCompiler import VariableCFF2sCompiler
+from ufo2ft._compilers.variableTTFsCompiler import VariableTTFsCompiler
+from ufo2ft.constants import CFFOptimization  # noqa: F401 (fontmake uses it)
```

Comment:
we could actually change fontmake to no longer do that, once we bump ufo2ft min requirements in fontmake, so I would not bother exporting this one